hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/checksum/ContainerChecksumTreeManager.java

```diff
@@ -65,8 +70,13 @@ public void writeContainerDataTree(KeyValueContainerData data, ContainerMerkleTr
       ContainerProtos.ContainerChecksumInfo newChecksumInfo = read(data).toBuilder()
           .setDataMerkleTree(tree.toProto())
           .build();
-      write(data, newChecksumInfo);
+      captureLatencyNs(metrics.getWriteContainerMerkleTreeLatencyNS(),
+          () -> write(data, newChecksumInfo));
```

Comment:
Should we also include the time to compute the checksums in this metric? That happens in `tree.toProto()` on line 71.

Reply:
Yes we should. I'll make sure to add them shortly.